presto-docs/src/main/sphinx/optimizer/statistics.rst

```diff
@@ -41,12 +41,13 @@ The following statistics are available in Presto:
    * **distinct value count**: the number of distinct values
```

Comment:
Please unindent this section to remove the vertical line in the left of this screenshot. 
<img width="819" alt="Screenshot 2024-06-20 at 3 04 43 PM" src="https://github.com/prestodb/presto/assets/7013443/b7919067-49be-4a63-ba3e-492b6e5ce0ba">